packages/bot-web-ui/src/components/dashboard/tutorial-tab/guide-content.tsx

```diff
@@ -112,7 +104,9 @@ const GuideContent = ({
                     has_close_icon
                     onClose={onOkButtonClick}
                 >
-                    {dialog_options.message}
+                    <React.Fragment>
```

Comment:
Why do we need a React.Fragment here? The Dialog component wraps <iframe>

Reply:
we don't need it. I will remove it.